src/component/signUp/SignUp.jsx

```diff
@@ -0,0 +1,95 @@
+import { useState, useContext } from 'react';
+import SimpleSnackbar from '../snack/Snack';
+import { Link, useNavigate } from 'react-router-dom';
+import { Context } from '../../index';
+import Header from '../header/Header';
+import Body from '../body/Body';
+import './SignUp.scss';
+import BodyImg from '../../img/hospital.svg';
+
+const SignUp = () => {  
+  const [newUser, setNewUser] = useState({
+    name: '',
+    password : '',
+    confirmPassword: '',
+  })
+  const [snackText, setSnackText] = useState('');
+  const [open, setOpen] = useState(false); 
+  const navigate = useNavigate();
+  const {store} = useContext(Context);  
+   
+  const createNewPatient = async (e) => { 
+    setOpen(true);
+    e.preventDefault();    
+    if (newUser.password === newUser.confirmPassword ) {
+      const res = store.registration(newUser.name, newUser.password);
+      res.then(value => (setSnackText(value)))      
+    }
+    else {
+      setSnackText(`password and confirm password are different!`);
+    } 
+  }  
+  
+  const handleChange = (key, value) => {
+    setNewUser({...newUser, [key]:value})
+  }
+
+  return (    
+    <form className='signUp' onSubmit={(e)=> createNewPatient(e)}>
+      <Header>
+        <div className='header-title'>
+          <h1>Зарегистрироваться в системе</h1>   
+        </div>               
+      </Header>
+      <Body>
+        <img src={BodyImg} alt='image'/>

```

Comment:
never call alt - image, the img tag already conveys the message that this is an image by default